Create job app

shayakrach opened the pull request:
Creating job model.
Migrating the jobs model.
Adding basic tests to the model.
Registering the jobs model to admin.

AshkenYariv reviewed (approved):
Looks good, @assafg95 will add the changes

assafg95 commented:
I will create tomorrow a new PR with the necessary changes. @AshkenYariv

shayakrach committed:
Create job app

Creating job model.
Adding basic tests to the model.
Migrating the jobs model.
Adding the jobs app to the list of apps Django knows about.
Registering the jobs model to admin.
